site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
+| ASP.NET Core App to Amazon ECS using Fargate               | Microsoft.NET.Sdk.Web                                                                  | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| ASP.NET Core App to AWS App Runner                         | Microsoft.NET.Sdk.Web                                                                  | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| Service on Amazon ECS using Fargate                        | Microsoft.NET.Sdk                                                                      | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| Scheduled Task on Amazon ECS using Fargate                 | Microsoft.NET.Sdk                                                                      | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| Blazor WebAssembly App                                     | Microsoft.NET.Sdk.BlazorWebAssembly (for.NET 5.0+) Microsoft.NET.Sdk.Web (for.NET 3.1) | netstandard2.1                               | No              |
+| ASP.NET Core App to AWS Elastic Beanstalk on Linux         | Microsoft.NET.Sdk.Web                                                                  | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | No              |
+| Push Container Images to Amazon Elastic Container Registry | N/A                                                                                    | N/A                                          | Yes             |
+
+You need to ensure that your .NET application falls into one of the categories in order to leverage AWS.Deploy.Tools
+
+Another reason why there are no recommendations generated is if your application `.csproj` file is using a variable for the `TargetFramework` property.
+
+
+    <Project Sdk="Microsoft.NET.Sdk.Web">
+	    <PropertyGroup>
+		    <TargetFrameworkVersion>net5.0</TargetFrameworkVersion>
+		    <TargetFramework>$(TargetFrameworkVersion)</TargetFramework>
+	    </PropertyGroup>
+	</Project>
+    
+No recommendations will be generated for the above `.csproj` file.
+**This is a bug which we will address.** Meanwhile, provide explicit values for the `TargetFramework` property.
+
+## Failed to create zip archive of the application
+Non-container based deployments types (such as deployments to AWS Elastic Beanstalk) create a zip file of the artifacts produced by the `dotnet publish` command.
+
+The zip command line utility is not installed by default on **Linux** based operating systems. If you are deploying using a non-container based option and encounter an error saying `failed to create a deployment bundle`, it is likely that zip is not installed on your system.
+
+To install zip on Linux OS, run the following command:
+```
+$ sudo apt-get install zip
```

Comment:
This is only valid on some Linux distributions. Others will need to do a `yum install`. Probably replace this with saying using your distribution's package management tools. For example distributions using `apt-get` you can use `sudo apt-get install zip`. For distributions using `yum` you can use `sudo yum intall zip`.

Reply:
And I'd also drop the leading `$` so that users can copy/paste the entire block as-is.